code/drasil-example/hghc/lib/Drasil/HGHC/Body.hs

```diff
@@ -70,9 +70,7 @@ ideaDicts =
   -- ConceptChunks
   map nw mathcon ++
   -- QuantityDicts
-  map nw symbols ++
-  -- UnitDefns
-  nw degree : map nw fundamentals ++ map nw derived
```

Comment:
The `symbMap` (a `ChunkDB`) already had the correct `siUnits` list of `UnitDefn`s used!